app/js/components/orderEntry/OrderEntryPage.jsx

```diff
@@ -38,12 +38,13 @@ export class OrderEntryPage extends React.Component {
           <div>
             <div className="error-notice">
               <p>
-                Setting for <strong>order.encounterType</strong> does not exist.
-                Please contact your administrator to create one for you.
+                Configuration for <strong>order.encounterType</strong> {error === 'incomplete config' ? 'is incomplete' : 'does not exist'}.
```

Comment:
great work, though I think that this error message can be improved.

Reply:
Thanks, please suggest how we can make this better

Reply:
sure, `
Creating new orders requires a configuration for an Encounter Type.  Please contact your administrator. 
For administrator.  The configuration process for  <strong>encounterType</strong> {error === 'incomplete config' ? 'is incomplete' : 'does not exist'}.
1. Please ensure that you have created an encounter type.
2. Ensure that you have also created a setting called <strong>order.encounterType</strong> with a value corresponding to the name of the encounter type created above.
` Iterartions are welcome.

Reply:
Well, as explanatory as this seems, I have been earlier warned against making error messages too specific because of security concerns. I guess we should speak with the team about this and get their opinions

Reply:
@larrystone decided on the error message to adopt?

Reply:
Awesome